docs/whats-new/preview-3.md

```diff
@@ -0,0 +1,446 @@
+---
+title: .NET Aspire preview 3
+description: .NET Aspire preview 3 is now available and includes many improvements and new capabilities
+ms.date: 02/13/2024
+---
+
+# .NET Aspire preview 3
+
+This article covers the 3rd preview of .NET Aspire. While the goal is to have monthly releases, January was skipped due to the holidays. .NET Aspire **preview 3** is now available!
+
+## Get .NET Aspire preview 3
+
+The complete set of [.NET Aspire setup and tooling](../fundamentals/setup-tooling.md) instructions has been updated to reflect preview 3. The following are the key steps to get started with preview 3:
+
+1. If on Windows and using Visual Studio to work with .NET Aspire, install/update to the latest [Visual Studio 2022 Preview](https://aka.ms/vspreview) release (17.10.0 Preview 1.0 at the time of writing).
+
+    - Ensure that the **.NET Aspire SDK (Preview)** component is selected under the **Individual components** tab of the Visual Studio installer:
+
+    ![Visual Studio installer with .NET Aspire SDK (Preview) component selected](media/preview-3/vs-aspire-sdk-component.png)
+
+1. If on Windows and you have Visual Studio 2022 17.8.x installed but intend to only use .NET Aspire via the .NET CLI (`dotnet`), download and install the [.NET 8.0.100 SDK using the standalone installer](https://dotnet.microsoft.com/download/dotnet/8.0).
+
+1. If on macOS or Linux, download and install the [.NET 8.0.100 SDK](https://dotnet.microsoft.com/download/dotnet/8.0).
```

Comment:
Maybe not important for a looser what's new page, but 1-3 might read better as a set of tabs for Visual Studio, CLI, and macOS/Linux with the workload instructions below it.

Reply:
I thought about that and had it that way originally. It felt a bit redundant with the changes to setup and tooling, which we link to - that have been updated. This was adapted from the original draft blog post, and I went with simplicity, and it felt like tabs was overkill.